src/plugin/oauth/index.js

```diff
@@ -1,14 +1,16 @@
 const { Assertion } = require("../../assertion");
 const { BasePlugin } = require("../../plugin");
-const { Issuer } = require("openid-client");
+const { Issuer, custom } = require("openid-client");
 const jwt = require("jsonwebtoken");
-const oauth2 = require("simple-oauth2");
 const queryString = require("query-string");
 const request = require("request");
 const URI = require("uri-js");
 
-Issuer.useRequest();
-Issuer.defaultHttpOptions = { timeout: 10000, headers: {} };
+custom.setHttpOptionsDefaults({
```

Comment:
hey, Travis, are these options configurable somewhere? We had some networking issues and wanted to test longer timeouts for our authentication backend

Reply:
Not currently but have thought about it. Not sure what makes the most sense to have a per-install defaults, or per-config defaults...

Reply:
I guess depends how much time you'd like to spend on that. Perhaps global default with per-config override? or default is already there, so maybe per-config is the right way to go, it won't be the most used feature anyway, just better for debugging or some edge cases I think